src/tests/integration/D3MAaveV3.t.sol

```diff
@@ -0,0 +1,1273 @@
+// SPDX-FileCopyrightText: © 2021 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity ^0.8.14;
+
+import {DssTest} from "dss-test/DssTest.sol";
+import "../interfaces/interfaces.sol";
+
+import { D3MHub } from "../../D3MHub.sol";
+import { D3MMom } from "../../D3MMom.sol";
+import { D3MOracle } from "../../D3MOracle.sol";
+
+import { D3MAavePlan } from "../../plans/D3MAavePlan.sol";
+import { D3MAavePool } from "../../pools/D3MAavePool.sol";
+
+interface Hevm {
+    function warp(uint256) external;
+    function store(address,bytes32,bytes32) external;
+    function load(address,bytes32) external view returns (bytes32);
+}
+
+// Need to use a struct as too many variables to return on the stack
+struct ReserveDataV3 {
```

Comment:
Also here, consider moving it inside the interface.

Reply:
[done](https://github.com/makerdao/dss-direct-deposit/pull/93/commits/a287a962595a66d2e4110d2bfb057c6b0643c1d7)